### PR TITLE
PayRue Space changes

### PR DESCRIPTION
**Pull request by AbstractBreazy**

### CODE REVIEW

File: spaces/payrue/index.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "name": "PayRue",
   "network": "56",
   "symbol": "Propel",
+  "skin": "payrue",
```
Comment (bonustrack) [inline spaces/payrue/index.json:6]: You can remove this as you don't have a custom theme
```suggestion
```

File: spaces/payrue/index.json [hunk 2]
```diff
@@ -13,8 +14,17 @@
...
   "filters": {
-    "defaultTab": "all",
-    "minScore": 0
-  }
-}
\ No newline at end of file
+    "defaultTab": "core",
+    "minScore": 1000000
+  },
+  "invalids": [
```
Comment (bonustrack) [inline spaces/payrue/index.json:24]: Invalids should be inside "filters"

**Review by gokaiorg — APPROVED** [review]

**Comment by gokaiorg** [comment]
We've made a major global update, please pull before commit again your changes. Thank you.

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**